grin/src/Pipeline/Pipeline.hs

```diff
@@ -1,20 +1,29 @@
-{-# LANGUAGE LambdaCase, RecordWildCards, TemplateHaskell, PatternSynonyms #-}
-module Pipeline.Pipeline where
+{-# LANGUAGE LambdaCase, RecordWildCards, RankNTypes #-}
+module Pipeline.Pipeline
+ ( module Pipeline.Pipeline 
+ , module Pipeline.Definitions
```

Comment:
As they reexposed via the same interface.

Reply:
See above comment.